packages/wasi/package.json

```diff
@@ -51,5 +50,12 @@
     "path-browserify": "^1.0.0",
     "randomfill": "^1.0.4"
   },
+  "devDependencies": {
```

Comment:
Can you add these to the root `package.json`?  This will help our install times, and will ensure that all projects have access to these for development 😄 

Also, will help not to fragment dev dependencies between packages

Reply:
Ah wait, Nevermind, seems like npx doesn't like it when it is not installed locally. But, actually, rollup works? Maybe its because you aren't running these commands from the root? 🤔